DailyReportWeb/Controllers/ReportController.cs

```diff
@@ -28,10 +36,53 @@ public ActionResult ResendDraft(string id, DateTime date)
             if (date.Date != DateTime.Today)
                 return Content("Cannot resend draft for another date");
 
-            if (ReportRunner.TryRunReport(id + "draft"))
+            if (ReportRunner.TryRunReport(id))
                 return Content("Draft report was resent");
-            else 
+            else
                 return Content("Error in resending draft report");
-        }     
+        }
+
+        [HttpGet]
+        public ActionResult SendIndividualDraft(string id, DateTime date, string draftKey)
```

Comment:
De fapt lasa e mai ok asa numele. Si hai sa redenumim dincolo SendDraft in loc de ResendDraft